src/popup/components/InputAmountV2.vue

```diff
@@ -4,6 +4,7 @@
     class="input-amount"
     type="number"
     placeholder="0.00"
+    inputmode="decimal"
```

Comment:
In my opinion this should be moved into InputField component and be based on input type.